runtime/j9vm/java11vmi.c

```diff
@@ -1890,3 +1890,15 @@ JVM_IsUseContainerSupport(void)
 
 	return result;
 }
+
+#if JAVA_SPEC_VERSION >= 17
+/**
+ * @brief Get the default system properties.
+ *
+ * @return an Object array of system property key value pairs
+ */
+jobjectArray JVM_GetProperties(JNIEnv *env)
```

Comment:
This should follow the pattern of other functions in this file:
```c
JNIEXPORT jobjectArray JNICALL
JVM_GetProperties(JNIEnv *env)
```

Reply:
Updated for both comments.